synthesizer/process/src/finalize.rs

```diff
@@ -104,10 +104,10 @@ impl<N: Network> Process<N> {
         lap!(timer, "Verify the number of transitions");
 
         // Construct the call graph.
-        // If the height is greater than or equal to `CONSENSUS_V3_HEIGHT`, then provide an empty call graph, as it is no longer used during finalization.
-        let call_graph = match state.block_height() < N::CONSENSUS_V3_HEIGHT {
-            true => self.construct_call_graph(execution)?,
-            false => HashMap::new(),
+        // If the height is greater than or equal to `ConsensusVersion::V3`, then provide an empty call graph, as it is no longer used during finalization.
+        let call_graph = match N::CONSENSUS_VERSION(state.block_height())? as usize {
+            1..3 => self.construct_call_graph(execution)?,
```

Comment:
```suggestion
            1..=2 => self.construct_call_graph(execution)?,
```
if possible - for ultimate clarity

Reply:
Good idea, 5b32053b7